libs/output-mapping/src/Storage/TableDataModifier.php

```diff
@@ -44,4 +39,29 @@ public function updateTableData(MappingFromProcessedConfiguration $source, Mappi
             }
         }
     }
+
+    private function prepareDeleteOptionsList(MappingFromProcessedConfiguration $source): array
```

Comment:
Podporujeme nový i starý formát. To že nejdou zadat současně je ošetřené v config definition